server/src/main/java/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -718,6 +718,10 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
         mailProps.setContent(content);
         mailProps.setContentType("text/plain");
 
+        if (recipients == null) {
+            return;
```

Comment:
Could you put a log message stating that the parameter was not set correctly?

Reply:
@RodrigoDLopez added an info log. Thanks